### PR TITLE
Changelog for v2.83.0

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,44 @@ Synapse Changelog
...
+====================
+
+Features and Enhancements
+-------------------------
+- Add ``:ip:proto`` and ``:ip:tcp:flags`` properties to the ``inet:flow``
+  form.
+  (`#2554 <https://github.com/vertexproject/synapse/pull/2554>`_)
+- Add ``$lib.log.debug()``, ``$lib.log.info()``, ``$lib.log.warning()``, and
+  ``$lib.log.error()`` Stormtypes APIs. These allow a user to send log
+  message to the Cortex output directly.
```
Comment (Cisphyx) [inline CHANGELOG.rst:18]: missing 's' on 'message'

File: CHANGELOG.rst [hunk 2]
```diff
@@ -5,6 +5,44 @@ Synapse Changelog
...
+- Add support for the ``synapse.tools.genpkg`` tool to support using files
```
Comment (Cisphyx) [inline CHANGELOG.rst:19]: The "add support to support" reads a little weird to me, maybe
"Update the ``synapse.tools.genpkg`` tool to support using files with the ``.storm`` extension"
or
"Add support for the ``synapse.tools.genpkg`` tool to use files with the ``.storm`` extension"

File: CHANGELOG.rst (hunk 1 above)
Comment (vEpiphyte) [reply CHANGELOG.rst:18]: fixed

File: CHANGELOG.rst (hunk 2 above)
Comment (vEpiphyte) [reply CHANGELOG.rst]: fixed

**Pull request merged**